### DISCUSSION TITLE
Question about jumpers

### DISCUSSION THREAD

**Discussion by pychneag** [thehighnibble/thehighnibble.github.io#41] [S-132 add-on for IMSAI & Cromemco replicas]
Hi. I'm working on my S-132. 

- I would like the S-132 to be powered from the connected IMSAI8080
- I'm using the PS/2 connector

I'm confused about the jumpers/patch header. Document states:

_**For JP1 & Patch as required | Header Pin Shunts**_

I cannot determine how to set: JP1 5V or the Header Pin Shunts for my use-case above. 

Thanks!

**Comment by dmcnaugh** [comment]
For the use case you describe you wont need any jumpers in place.

1. The S-132 is already connected to the IMSAI8080 for power
2. Communications between the IMSAI8080 and the S-132 doesn't require any jumpers 

Note:
* JP1 is a bypass for the LM7805
  * you only fit this if you are using a **+5VDC** supply plugged into the DC Barrel Jack on the S-132
* The `Patch` header (4x2) on the S-132 simply reproduces the 8 pins of the original IMSAI `Patch` header used for patching Rx & Tx (1 &2) to the MAX3232 for the RS-232 DE-9 connectors

**Reply by cavaughankirov** [reply]
I'm sorry I'm confused about the JP1. dmcnaugh seems to be saying it should have a shunt on it if one is using the barrel jack? But in every pic on The High Nib, that header is not shunted. I have not shunted it, but when I plug it in with a 7.5 V supply, the IMSAI is not getting any power. Also, it seems I could just use the 5v USB power supply that I use for the IMSAI. But that definitely isn't working when the S132 is plugged in. The IMSAI doesn't fire up. So I'm thinking I have a short or something in the S132.

**Reply by dmcnaugh** [reply]
As you are using a +7.5VDC supply you should not fit `JP1`.
As mentioned: `JP1` bypasses the **LM7805** regulator and should only be fitted if you are supplying +5VDC to the DC Barrel Jack.

With your +7.5VDC supply connected, you can then use a multimeter to see if you are getting +5V across the board and up to the `S-132` header (8x2 or 16pin header) that connects to the IMSAI.

**Reply by cavaughankirov** [reply]
Okay, figuring this out. So I see that it's 7.5 at the source, but at the header there is no power.

**Reply by dmcnaugh** [reply]
What about on the output of the LM7805? That's the source of +5V for the rest of the board.

**Reply by cavaughankirov** [reply]
Ok, now I'm confused... Such a neophyte.... If I test directly on the output plug, I get 7.5. But  if I test on the soldered sections to the plug, I'm only getting like just over 5v. Now only one of the LM7805 shows power and it's like 1v. And now it's hot as hell. Just burnt myself.....

**Reply by cavaughankirov** [reply]
This is weird. So I unplugged it. Let the LM7805 cool down. Plugged it back in, nothing across the LM7805. Actually, sometimes I get a slight movement, but then nothing. Ok, yeah, it's acting like a capacitor. Waited again. Touched a lead to the LM7805, got voltage, but dropped somewhat slowly to nothing. And it doesn't seem to be getting hot.

**Reply by cavaughankirov** [reply]
Ok, scratch that..... I unplugged it. Let it sit and plugged it back in. Now that the LM7805 is boiling hot, I know it's doing something. So using the soldered posts on the underside on the one end it's the same 5v, as for the power source, but on the other end it's only around 1v.
